client/src/components/Owner/OwnerContainer.jsx

```diff
@@ -1,16 +1,30 @@
 import useEth from "../../contexts/EthContext/useEth";
 import AddVoter from "./AddVoter";
+import { useState } from "react";
+
 
 const OwnerContainer = () => {
   const { state: { accounts, contract } } = useEth();
+  const [error, setError] = useState()
+
+  const StartProposal = async e => {
```

Comment:
@rembeuch 
Comme tu utilises `async` & `await` le code devrait être plutôt, comme cela :
```suggestion
  const startProposal = async () => {
    try {
      await contract.methods.startProposalsRegistering().send({ from: accounts[0] });
    } catch (error) {
      setError(error.message.match(/revert (.*)/)[1]);
    }
  };
```
Ou version avec des promises 

```suggestion
  const startProposal = () => {
    contract.methods.startProposalsRegistering().call({ from: accounts[0] })
      .then(result => {
        return contract.methods.startProposalsRegistering().send({ from: accounts[0] });
      })
      .catch(error => {
        setError(error.message.match(/revert (.*)/)[1]);
      });
  };
```

Je pense que la version 1 est plus lisible et propre.

Dans ta version, tu mélanges `await` avec `callback` et `function` avec `arrow function`.
:v:

Reply:
il faut malheureusement utiliser la 2ème méthode car sans le .call on ne récupère pas l'erreur du require.